lang/cs/Org.Apache.REEF.IMRU/API/IMRUJobDefinitionBuilder.cs

```diff
@@ -203,6 +211,17 @@ public IMRUJobDefinitionBuilder SetUpdateTaskMemory(int memory)
         }
 
         /// <summary>
+        /// Sets Per Map Configuration
```

Comment:
Just trying to understand , you want to have ISet<IPerMapperConfig> ??  I
am a little confused what use will it have? Since user can always merge and
set it as single configuration or am I missing something?

Dhruv

On Fri, Aug 28, 2015 at 7:38 PM, Markus Weimer <notifications@github.com>
wrote:

> In lang/cs/Org.Apache.REEF.IMRU/API/IMRUJobDefinitionBuilder.cs
> <https://github.com/apache/incubator-reef/pull/417#discussion_r38258068>:
>
> > @@ -203,6 +211,17 @@ public IMRUJobDefinitionBuilder SetUpdateTaskMemory(int memory)
> >          }
> >
> >          /// <summary>
> > +        /// Sets Per Map Configuration
>
> I think we should generalize this a bit. How about allowing for arbitrary
> additional IConfiguration instances to be added to the Driver? Then, this
> would just be one special case of that generic mechanism.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/incubator-reef/pull/417/files#r38258068>.
>